src/styles/main.scss

```diff
@@ -3,12 +3,28 @@
 
 #root {
     @import "containers/home";
+    background: $light-sea-green;
+}
+
+#outer-container {
```

Comment:
I didn't actually realize this file was update in this PR. I'll give it a look over myself. ;)